package.json

```diff
@@ -166,6 +166,7 @@
   },
   "scripts": {
     "dev": "concurrently --kill-others -p name -n 'backend,frontend,docs' -c 'blue,green,yellow' 'lein run' 'yarn build-hot' 'yarn docs'",
+    "dev-ee": "concurrently --kill-others -p name -n 'backend,frontend,docs' -c 'blue,green,yellow' 'lein with-profiles +ee run' 'yarn build-hot' 'yarn docs'",
```

Comment:
I think this should set `MB_EDITION` for `yarn build-hot`

Reply:
What do you mean? Isn't `MB_EDITION` ENV we're setting manually?

Reply:
Excellent suggestion! Thanks :)